docs/compilation.rst

```diff
@@ -339,12 +339,21 @@ Cross-compiling DALI C++ API for aarch64 Linux (Docker)
   x86-64 target and they are turned off in this build. There is no support for DALI Python library
   on aarch64 yet. Some Operators may not work as intended due to x86-64 specific implementations.
 
+Setup
+^^^^^
+Using the SDK Manager for NVIDIA Jetson as the product category adn JetPack 4.4.
+Then select CUDA for the host. After download process has been completed move
+``cuda-repo-ubuntu1804-10-2-local-10.2.89-440.40_1.0-1_amd64.deb`` and
+``cuda-repo-cross-aarch64-10-2-local-10.2.89_1.0-1_all.deb`` from the download folder
+to main DALI folder.
```

Comment:
Maybe mention that they are reqired for the cross-build?

Reply:
Done